src/script/auth/page/Login.tsx

```diff
@@ -161,8 +161,12 @@ const Login = ({
   const immediateLogin = async () => {
     try {
       await doInit({isImmediateLogin: true, shouldValidateLocalClient: false});
+      // Try to initialize to check if the client needs 2fa before checking for entropyData
+      await doInitializeClient(ClientType.PERMANENT, undefined, undefined, undefined);
```

Comment:
Hummm this feels wrong 🤔  I guess you could end-up creating a device that has no-custom entropy (while it's needed)

Reply:
could you? it should recreate it if its needed no?